flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointStatsTracker.java

```diff
@@ -327,6 +327,9 @@ public void reportIncompleteStats(
     static final String LATEST_COMPLETED_CHECKPOINT_EXTERNAL_PATH_METRIC =
             "lastCheckpointExternalPath";
 
+    @VisibleForTesting
+    static final String LATEST_COMPLETED_CHECKPOINT_ID_METRIC = "lastCheckpointId";
```

Comment:
"lastCheckpointId" ->"lastCompletedCheckpointId"

lastCheckpointId is different from lastCompletedCheckpointId

Reply:
Also, please squash the commits, and I will do a final round.

Reply:
To clarify why it has to specify "completed": failed cp also has an id, `lastCheckpointId` can refer to a failed cp id, not necessarily a successful id. 

For other metrics, like full size or duration, it mostly make sense for successful checkpoints.

Reply:
Thanks for your advice! I have changed it